superset-frontend/cypress-base/cypress/integration/dashboard/save.test.js

```diff
@@ -50,21 +52,32 @@ describe('Dashboard save action', () => {
 
   it('should save/overwrite dashboard', () => {
```

Comment:
This test seems to be failing on CI:
![Uploading image.png…]()

Reply:
Fixed :)